feincms3_sites/__init__.py

```diff
@@ -1,4 +1,24 @@
+from django.apps import apps as django_apps
+from django.conf import settings
+from django.core.exceptions import ImproperlyConfigured
```

Comment:
Can you move those imports into the `get_site_model()` function? My (possibly archaic) packaging workflow still requires that `setup.py` is allowed to import `feincms3_sites/__init__.py`

Reply:
Uh, I took that over from auth and tried out different things including also having it in `feincms3_sites.models`. Will move it back to `feincms3_sites.models`.

`get_site_model` is a convenience function for users who want to associate custom (app) content with specific sites.

Inside `feincms3_sites.model` we cannot use that function as it will fail since the other app and its models may not be initialized, hence the signal handler in `feincms3_sites.models` for the `AbstractPage`.

Reply:
models are partially initialized, will have to disect the models module into a models package

Reply:
i had to move `get_site_model` to `models/utils.py` to overcome circular depencendies and also premature import of otherwise uninitialized/unprepared models.

Reply:
I'd rather have the parts be in one file than introduce a models package if it is possible. This was just a suggestion, I'm sorry for the trouble it caused 😅

Reply:
hm, get_site_model is used by the middleware, which cannot include the models as they are only partially initialized at the time that the middleware is realized.
so i need to move that function out of the models in order to make the models a single package again, say `feincms3_sites.utils`.

Reply:
refactored.

Reply:
but not tested yet :-)

Reply:
well. it works...